app/models/application_record.rb

```diff
@@ -1,5 +1,5 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
 
-  default_scope { order(created_at: :desc) }
+  default_scope { order(created_at: :desc) } # TODO : Remove this.
```

Comment:
I removed it already and replaced it with a service to order the model, can you rebase the branch to master? as also there is a conflict.